### DISCUSSION TITLE
Gravity Form Display Issue

### DISCUSSION THREAD

**Discussion by tomthor** [WebDevStudios/nextjs-wordpress-starter#832] [Q&A]
I'm following the documentation on how to display the gravity forms but when I add a form on a page I get a 404 error. 

The documentation is a little unclear in regards to the setup: 

<img width="722" alt="Screenshot 2022-01-01 at 16 33 25" src="https://user-images.githubusercontent.com/17763837/147851788-ac069ac3-b18c-4cb6-b3f4-9dff06b148ce.png">

It says to "To display a Gravity Form, simply use the Gravity Form block" - Does this mean use the Gutenberg Gravity Form block in the Wordpress/Gutenberg editor? Or does this mean use the Component block in the code? 

Checking the starter demo site it looks like it's been added in the page content via a Gutenberg block (I could be wrong?). It would make sense to have it syncing via a Gutenberg block so a form can be created on the fly and added wherever.

Or is a Gravity Form only used as a component? Added via a custom page template?

**Comment by gregrickaby** [comment]
@tomthor Yeah, I can see how this is ambiguous. It should read, "simply use the Gravity From block **inside the WordPress post editor**". 

I'll get this fixed up. Sorry for the confusion.
